docs/services/slam/cartographer/_index.md

```diff
@@ -54,188 +81,380 @@ Follow the instructions below to set up the `cartographer` module on your robot:
+| `enable_mapping` | boolean | Optional | If `true`, Cartographer will build the map in addition to doing localization. <ul> Default: `true` </ul> |
+| `existing_map` | string | Optional | The alias of the package containing the existing map to build on (in "Update existing map" mode) or localize on (in "Localize only" mode). |
+| `config_params` |  obj | Optional | Parameters available to fine-tune the `cartographer` algorithm: [read more below](#config_params). |
 
-### Adjust `data_dir`
+### View the Map
 
-Change the `data_dir` attribute to point to a directory on your machine where you want to save the internal state your SLAM service produces.
+#### "Create new map" mode
 
-This directory must be structured as follows:
+Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
 
-<pre>
-.
-└──\(<file>CARTOGRAPHER_DIR</file>)
-    ├── <file>internal_state</file>
-</pre>
+Enter a name for your new map and click `"Start session"`, then wait for the slam session to finish starting up in the cloud, which **takes about 2 minutes**.
 
-The SLAM Mapping Mode is determined by 2 conditions:
+Make sure to either **manually refresh**, or **change the refresh frequency** to something other than `Manual`.
 
-1. If the internal state data is present in <file>internal_state</file> at runtime
-2. The attribute `map_rate_sec`
+![slam RC card start session](/services/slam/slam-RC-card-start-session.png)
 
-### SLAM Mapping Modes
+While the slam session is starting, you will see a loading screen.
 
-<!-- prettier-ignore -->
-| Mode | Description | Runtime Dictation |
-| ---- | ----------- | ------- |
-| PURE MAPPING | Generate a new internal state in <file>/internal_state</file>. | No internal state is found in <file>/internal_state</file> + [`map_rate_sec > 0`](#attributes). |
-| UPDATING | Update an existing internal state with new sensor readings. | An existing internal state is found in <file>/internal_state</file> + [`map_rate_sec > 0`](#attributes).|
-| LOCALIZING | Localize the robot on an existing internal state without changing the internal state itself. | An internal state is found in <file>/internal_state</file> + [`map_rate_sec = 0`](#attributes). |
+![slam RC card wait for session to finish starting](/services/slam/slam-RC-card-wait-for-session-to-finish-starting.png)
+
+Once the slam session has finished starting, your first pointcloud will appear.
+
+![slam RC card first pointcloud](/services/slam/slam-RC-card-first-pointcloud.png)
+
+You will be able to see that your cloud slam session is in progress from your **Location** page's **SLAM library** tab.
+
+![offline mapping maps computing table](/services/slam/offline-mapping-maps-computing-table.png)
+
+When you would like to end the slam session, back on the robot's **Control** tab, click `"End session"`. If you do not click `"End session"`, the slam session will automatically be ended after 45 minutes.
```

Comment:
```suggestion
When you would like to end the slam session, return to your robot's **Control** tab, and click `"End session"`.
If you do not click `"End session"`, the slam session will automatically be ended after 45 minutes.
```

Reply:
Question: "after 45 minutes" of inactivity? Of no data received by cloud compute? Or is it a hardcoded 45m elapsed since start regardless?

Reply:
hardcoded, though its actually hardcoded to 50 minutes in the backend